docs/howto.md

```diff
@@ -0,0 +1,232 @@
+# How to
+
+## Add A Functional Zone
```

Comment:
So I was thinking about that, but this extends just the scope of a simple Primary Navigation entry (part of a larger super set of functionality)